spy/cli.py

```diff
@@ -55,6 +55,14 @@ class Arguments:
         )
     ] = False
 
+    colorize: Annotated[
+        bool,
+        Option(
+            "-C", "--colorize",
+            help="Output the pre-redshifted AST with blue / red text colors."
+        )
+    ] = False
```

Comment:
`-C` is already used as an alias for `--cwrite`.
I think that `--colorize` is more important, so +1 to use `-C` here, but we should remove `-C` from the `cwrite` option.